thumbnail/README.md

```diff
@@ -298,8 +298,8 @@ access to your COS buckets is restricted and can not be accessed by any other
 cloud components without your explicit permission. In this case, we need to
 authorize access to your COS bucket from your Code Engine project.
 
-To do this we'll first need the Service Instance ID of our Code Engine project.
-To get that we'll use the following command:
+Since Code Engine itself also behaves like an Service Offering and inherited projects like Service Instances of this, we can use the Service Instance ID of our Code Engine project to accomplish this.
```

Comment:
I'm having trouble parsing this ;-)
Perhaps:
To establish this authorization you'll need the ID of the Code Engine project that receive the COS related events.